src/main/java/fi/crosskey/card/vehicle/simulator/service/HelicoptersController.java

```diff
@@ -0,0 +1,42 @@
+package fi.crosskey.card.vehicle.simulator.service;
+
+
+import fi.crosskey.card.vehicle.simulator.models.Helicopter;
+import fi.crosskey.card.vehicle.simulator.repositories.HelicopterRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+
+@RestController
+@RequestMapping("/helicopters")
+public class HelicoptersController {
+
+    @Autowired
+    private HelicopterRepository vehiclesRepository;
+
+    public HelicoptersController(final HelicopterRepository vehiclesRepository) {
+        this.vehiclesRepository = vehiclesRepository;
+    }
+
+    @RequestMapping(method = RequestMethod.GET)
+    public Iterable<Helicopter> findAll() {
+        return vehiclesRepository.findAll();
+    }
+
+    @RequestMapping(method = RequestMethod.POST)
+    public  <S extends Helicopter> void save(S entity) {
+        vehiclesRepository.save(entity);
+    }
+
+    /* URL path = localhost:8080/helicopters/findOne/value */
+
+    @RequestMapping(value = "/findOne/{id}", method = RequestMethod.GET)
```

Comment:
So, what we're building here is a RESTful webservice for helicopters.

If you have time, read this (or other resources) about designing REST services: https://openclassrooms.com/en/courses/3432056-build-your-web-projects-with-rest-apis/3496011-examples-of-rest-apis

Did you choose an appropriate URL for this endpoint?